app/controllers/deploys_controller.rb

```diff
@@ -1,18 +1,13 @@
 class DeploysController < ApplicationController
-  include CurrentProject
   include ProjectLevelAuthorization
 
-
-  before_action except: [:active, :active_count, :recent, :changeset] do
-    find_project(params[:project_id])
-  end
-
```

Comment:
This refactoring has the ultimate goal of removing this before action. In the code review from pull request #553, It was suggested that I could remove this before_actions in favour of a lazy initialised method (current_project).

Despite knowing I would end up with this repeated code, the reason why I added a line in each action where we need to initialise @project was, first, to make it clear where we need it to be initialised, and second, to minimise the impact from my changes and the size of this PR.

But there's a way to prevent the code duplication and still get rid of the before_action, so, what do you think about:
- Removing the before_actions, but...
- In all the actions where we need the @project, instead of initialising it explicitly, just call the current_project method?

Example:
```
def index
  @project = current_project
  @deploys = @project.deploys.page(..)
end
```
would end up as:
```
def index
  @deploys = current_project.deploys.page(..)
end
```